Sources/SmileID/Classes/SelfieCapture/SelfieViewModel.swift

```diff
@@ -314,20 +305,59 @@ public class SelfieViewModel: ObservableObject, ARKitSmileDelegate {
                     )
                 }
                 let authResponse = try await SmileID.api.authenticate(request: authRequest).async()
-                let prepUploadRequest = PrepUploadRequest(
-                    partnerParams: authResponse.partnerParams.copy(extras: extraPartnerParams),
-                    allowNewEnroll: String(allowNewEnroll),
-                    timestamp: authResponse.timestamp,
-                    signature: authResponse.signature
-                )
-                let prepUploadResponse = try await SmileID.api.prepUpload(
-                    request: prepUploadRequest
-                ).async()
-                _ = try await SmileID.api.upload(
-                    zip: zip,
-                    to: prepUploadResponse.uploadUrl
-                ).async()
-                didSubmitSmartSelfieJob = true
+
+                var smartSelfieLivenessImages = [SmartSelfieRequestImage]()
+                var smartSelfieImage: SmartSelfieRequestImage?
+                if let selfie = try? Data(contentsOf: selfieImage), let media = SmartSelfieRequestImage(
+                    withImage: selfie,
+                    forKey: selfieImage.lastPathComponent,
+                    forName: selfieImage.lastPathComponent
+                ) {
+                    smartSelfieImage = media
+                }
+                if !livenessImages.isEmpty {
+                    let livenessImageInfos = livenessImages.compactMap { liveness -> SmartSelfieRequestImage? in
+                        if let data = try? Data(contentsOf: liveness) {
+                            return SmartSelfieRequestImage(
+                                withImage: data,
+                                forKey: liveness.lastPathComponent,
+                                forName: liveness.lastPathComponent
+                            )
+                        }
+                        return nil
+                    }
+
+                    smartSelfieLivenessImages.append(contentsOf: livenessImageInfos.compactMap { $0 })
+                }
+                // swiftlint:disable line_length
+                guard let smartSelfieImage = smartSelfieImage, smartSelfieLivenessImages.count == numLivenessImages else {
```

Comment:
Instead of disabling this, you should be able to do something like the below (or similar):
```swift
guard let smartSelfieImage = smartSelfieImage, 
smartSelfieLivenessImages.count == numLivenessImages else {
```